scripts/src/metrics/metrics.py

```diff
@@ -330,17 +337,20 @@ def send_check_metric(write_key,type,partner,chart,pr_number,check):
 
 def send_merge_metric(write_key,type,partner,chart,duration,pr_number,num_builds,pr_content):
 
-    id = f"helm-metric-{partner}"
+    #id = f"helm-metric-{partner}"
+    id = f"{id_prefix}-{type}-{partner}"
     properties = { "type" : type, "provider": partner, "chart" : chart, "pr" : pr_number, "builds" :num_builds, "duration" : duration, "content" : pr_content}
```

Comment:
Thinking it would be good to add the update metric here too.

Reply:
yeah makes sense